README.md

```diff
@@ -1,222 +1,134 @@
-# BADGES
 
+[![Issues](https://img.shields.io/github/issues/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/issues)
+[![PRs](https://img.shields.io/github/issues-pr/praveenscience/Internship-LMS-FrontEnd)](https://github.com/praveenscience/Internship-LMS-FrontEnd/pulls)
+[![Maintenance](https://img.shields.io/maintenance/yes/2020?color=green&logo=github)](https://github.com/praveenscience/)
+[![Forks](https://img.shields.io/github/forks/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Stars](https://img.shields.io/github/stars/praveenscience/Internship-LMS-FrontEnd?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd) 
+[![Watchers](https://img.shields.io/github/watchers/praveenscience/foss-events?style=social)](https://github.com/praveenscience/Internship-LMS-FrontEnd)
 
 # Introduction
 
- 
-
-
- # Motivation
-
-
-
+ We are in the mission of solving a crisis for a lot of small and medium enterprises, who are trying to recruit interns for their projects. This platform aims to onboard new interns, who have applied to the companies and help them understand the process and make the transition smooth.
 
- # Methodology
-As this project is not your usual Open Source project, this is not going be a bug fixing or a robotic one. Please read the messages carefully and let's work as a team. We will be following Agile Methodology, specifically [Spotify Agile Model](https://rb.gy/tzvtsj) with some [Coding Standards](https://rb.gy/jmo3xp) being followed, which I'll discuss later.
+- Motivation
+  - Atuomate the process of regular tasks.
+  - Make the task easy and stress free of managment members.
+  - Save the time of people in doing same work multiple times.
 
+- Methodology
 
+- Video Links
+  - [Intro Video](https://rb.gy/cygmlf)
+  - [GSSoC 21 Series](https://rb.gy/o7lwj8)
+  - [BrainStorming Day 1: Project Plan](https://rb.gy/dfxjnd)
+  - BrainStorming Day 2: Scrum Masters (TBD)
 
- # Video Links
-
-Intro Video (https://rb.gy/cygmlf)
-GSSoC 21 Series (https://rb.gy/o7lwj8)
-BrainStorming Day 1: Project Plan (https://rb.gy/dfxjnd)
-BrainStorming Day 2: Scrum Masters (TBD)
-
-
-
-# Resources
-
-OpenSource Intro (https://rb.gy/dxnv5y)
-30 Days of React (Event: https://rb.gy/zsgyja, PlayList: https://rb.gy/cxk3ek)
-
-
+- Resources
+  - [OpenSource Intro](https://rb.gy/dxnv5y)
+  - 30 Days of React
+    - [Event](https://rb.gy/zsgyja)
+    - [PlayList](https://rb.gy/cxk3ek)
 
 # GSSoC Participation
+![](https://i.imgur.com/BR9Q5Pd.png)
+  - Rules of GSSoC 21
+  - Levels & Points
+
+<table style="width:100%">
+  <tr>
+    <th>Level</th>
+    <th>Points</th> 
+  </tr>
+  <tr>
+    <td>Level 0</td>
+    <td>5</td>
+    
+  </tr>
+  <tr>
+    <td>Level 1</td>
+    <td>10</td>
+    
+  </tr>
+  <tr>
+    <td>Level 2</td>
+    <td>15</td>
+    
+  </tr>
+  <tr>
+    <td>Level 3</td>
+    <td>25</td>
+    
+  </tr>
+</table>
```

Comment:
Remove all the empty lines here. Check if these scores are right. Where did you get them from?